Automattic/jetpack
# Search: Add a function to estimate record counts pre-connection

Fixes #23239.

#### Changes proposed in this Pull Request:

Adds a record count estimation function.





#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:





* Apply this change to your **local, disconnected** WordPress site.
* Revert aa01c24eeaaac9e876dfdd763685b219e9a7c6f6 to re-enable testing code.
* With the Jetpack Search plugin enabled, navigate to the Jetpack Search Dashboard.
* Ensure you get a console log with a record count estimate.
